codebuild/create_batch.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -eu
+
+BUILDSPEC_OMNIBUS=./spec/buildspec_omnibus.yml
+BUILDSPEC_FUZZ=./spec/buildspec_fuzz_batch.yml
+BUILDSPEC_INTEG=./spec/buildspec_integ_batch.yml
+BUILDSPEC_GENERAL=./spec/buildspec_general_batch.yml
+
+synth_subjobs () {
+  yq -S -Y -r '{batch:{"build-list":[.batch."build-list"[]| select(.identifier|contains("Fuzz")) ]}}' $BUILDSPEC_OMNIBUS > $BUILDSPEC_FUZZ
+  yq -S -Y -r '{batch:{"build-list":[.batch."build-list"[]| select(.identifier|contains("Integ")) ]}}' $BUILDSPEC_OMNIBUS > $BUILDSPEC_INTEG
+  yq -S -Y -r '{batch:{"build-list":[.batch."build-list"[]| select(.identifier|contains("Fuzz")|not)|select(.identifier|contains("Integ")|not) ]}}' $BUILDSPEC_OMNIBUS > $BUILDSPEC_GENERAL
+}
```

Comment:
I'm still worried that this isn't enforced. Maybe it should be called as part of create_project.py and we shouldn't commit the generated files?

Reply:
For now we need it in the repository.  The other option for build-spec files is storing them in-line in the CodeBuild job, which would allow us to save it at job creation time (which isn't ready yet from troposphere).  The counter to this is that anyone can go in and modify these jobs in the console, so without enforcing drift with CloudFormation, we currently aren't using a mechanism to prevent this

Reply:
reworking this a bit- updated the description.

Reply:
It looks like you're still committing the generated files. Shouldn't they not be committed, and possibly added to gitignore just in case?

Reply:
removed and gitignored